source/features/file-finder-buffer.tsx

```diff
@@ -14,8 +14,8 @@ const getBufferField = onetime((): HTMLInputElement => (
 ));
 
 function pjaxStartHandler(event: CustomEvent): void {
-	const destinationURL = event.detail?.url || '';
-	if (destinationURL.split('/')[5] !== 'find') {
+	const destinationURL = new URL(event.detail?.url || location.href);
+	if (!pageDetect.isFileFinder(destinationURL)) {
```

Comment:
More like:
```suggestion
	const destinationURL = event.detail?.url;
	if (!destinationURL || !pageDetect.isFileFinder(new URL(destinationURL))) {
```

This skips the testing altogether when missing